include/abb_librws/rws_rapid.h

```diff
@@ -371,6 +372,243 @@ struct RAPIDRecord : public RAPIDSymbolDataAbstract
   std::vector<RAPIDSymbolDataAbstract*> components_;
 };
 
+/**
+ * \brief A struct, for representing the data of a RAPID array of atomic type.
+ */
+template<typename T>
+struct RAPIDArray : public RAPIDSymbolDataAbstract
```

Comment:
1. Consider inheriting from the container type.
2. Is the array size a part of the type? Consider making array size the second parameter of the template.